condensation/scripts/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2015 Mirantis Inc.
```

Comment:
There is no need to put copyrights to the `__init__.py` files, so it can be removed as well.